src/app/projects/page.tsx

```diff
@@ -0,0 +1,130 @@
+import { Card } from "@/components/Card";
+import { SimpleLayout } from "@/components/SimpleLayout";
+import { Project } from "@/interfaces/project";
+import { getAllProjects } from "@/lib/api";
+
+function ProjectStatus({ status }: { status: Project['status'] }) {
+  const statusConfig = {
+    'done': { label: 'Completed', className: 'bg-green-900/50 text-green-300 border-green-500/50' },
+    'in-progress': { label: 'In Progress', className: 'bg-blue-900/50 text-blue-300 border-blue-500/50' },
+    'abandoned': { label: 'Archived', className: 'bg-zinc-700/50 text-zinc-400 border-zinc-600/50' },
+  };
+
+  const config = statusConfig[status];
+  
+  return (
+    <span
+      className={`border px-2 py-0.5 rounded text-xs font-medium ${config.className}`}
+    >
+      {config.label}
+    </span>
+  );
+}
+
+function ProjectLinks({ project }: { project: Project }) {
+  const links = [];
+  
+  if (project.github_link) {
+    links.push({ url: project.github_link, label: 'GitHub', type: 'github' });
+  }
+  if (project.demo_link) {
+    links.push({ url: project.demo_link, label: 'Demo', type: 'demo' });
+  }
+  if (project.youtube_link) {
+    links.push({ url: project.youtube_link, label: 'YouTube', type: 'youtube' });
+  }
```

Comment:
There will be no more youtube links. Those were entered manually. We don't do that anymore